sdk/tests/deqp/functional/gles3/es3fFramebufferBlitTests.js

```diff
@@ -726,7 +726,7 @@ goog.scope(function() {
         for (var ndx = 0; ndx < 2; ndx++) {
             ctx.bindFramebuffer(gl.FRAMEBUFFER, ndx ? dstFbo : srcFbo);
             if (ndx) {
-                gradientToDstShader.setGradient(ctx, gradShaderDstID, dstRangeInfo.valueMax, dstRangeInfo.valueMin);
+                gradientToDstShader.setGradient(ctx, gradShaderDstID, dstRangeInfo.valueMin, dstRangeInfo.valueMax);
                 rrUtil.drawQuad(ctx, gradShaderDstID, [-1, -1, 0], [1, 1, 0]);
             } else {
                 gradientToSrcShader.setGradient(ctx, gradShaderSrcID, srcRangeInfo.valueMin, srcRangeInfo.valueMax);
```

Comment:
This line isn't consistent with C++ code either.  Don't know the reason. dstRangeInfo.valueMax and srcRangeInfo.valueMax have the same value in this case.
https://android.googlesource.com/platform/external/deqp/+/master/modules/gles3/functional/es3fFramebufferBlitTests.cpp#409

Don't know why C++ code reverse valueMin and valueMax order. I think initialized data in dst  framebuffer doesn't take effect to the blit result.